trunk/SUAVE/Methods/Weights/Correlations/Tube_Wing/operating_items.py

```diff
@@ -0,0 +1,82 @@
+## @ingroup Methods-Weights-Correlations-Tube_Wing
+# operating_items.py
+#
+# Created:  Jan 2014, A. Wendorff
+# Modified: Feb 2014, A. Wendorff
+#           Feb 2016, E. Botero
+#           May 2020, W. Van Gijseghem
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+from SUAVE.Core import Units, Data
+import numpy as np
+
+
+def operating_system(vehicle):
+    """ Calculate the weight of operating items, including:
+        - crew
+        - baggage
+        - unusable fuel
+        - engine oil
+        - passenger service
+        - ammunition and non-fixed weapons
+        - cargo containers
+
+        Assumptions:
+
+        Source:
+            http://aerodesign.stanford.edu/aircraftdesign/AircraftDesign.html
```

Comment:
Looks like the wrong source here.

Reply:
I am not sure what you mean. These operating items weights are taken from Kroo's method presented on http://aerodesign.stanford.edu/aircraftdesign/AircraftDesign.html

Reply:
The all caps variables are to be consistent with the FLOPS method. Let me know if you want this to change

Reply:
I'd suggested writing out the variables if they're general purpose, but it would be helpful to callout their FLOPS equivalent in comments.

Reply:
Changed in this file, I suspect you don't want me to change it in the FLOPS files correct?